server/src/instant/dash/routes.clj

```diff
@@ -1209,6 +1219,7 @@
 
   (POST "/dash/apps/:app_id/rename" [] app-rename-post)
 
+  (GET "/dash/apps/:app_id/storage/enabled" [] storage-enabled-get)
```

Comment:
nit: perhaps we should include this in the `dash-get` route? 

This way, there would be no loading screen when the user clicks 'storage'.

Reply:
good idea!